curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/664b82025b371a24781fd40a.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 664b82025b371a24781fd40a
+title: Step 1
+challengeType: 1
+dashedName: step-1
+---
+
+# --description--
+
+JavaScript is the programming language that powers the web. Unlike the HTML and CSS you have learned previously, JavaScript is most commonly used to write logic instead of markup.
+
+In this project, you will learn the basics of Javascript and apply those concepts to building a pyramid generator. 
+
+A pyramid generator is a program where you can set the type of character, the count for the pyramid, and the direction of the pyramid. The program will then generate a pyramid based on those inputs.
+
+Remove all of the code in the editor and click on the `"Check your code"` button to proceed to the next step and start the project. 
+
+# --hints--
+
+There should be no code present in the editor. 
+
+```js
+assert.isEmpty(editableContents);
```

Comment:
```suggestion
assert.strictEqual(code.trim(), "");
```

We can assert that there's nothing here at all.

Reply:
The test can be edited to account for that, then. :3

Reply:
The editableContents being empty does seem clearer though?